package main

func main() {
	panic("This is a error!")
}
